docroot/themes/custom/mass_admin_theme/templates/form/form-element.html.twig

```diff
@@ -63,8 +63,11 @@
     description_display == 'invisible' ? 'visually-hidden',
   ]
 %}
-
-{% set id = name|clean_id %}
+{% if name is not empty %}
+  {% set id = name|clean_id %}
+{% else %}
+  {% set id = random() %}
```

Comment:
Does random() respect the valid character rules for an id? See https://www.w3.org/TR/html4/types.html#:~:text=ID%20and%20NAME%20tokens%20must,periods%20(%22.%22).

Reply:
Moshe and Arthur discussed on standup and Moshe approves this.